src/main/java/no/fdk/referencedata/geonorge/administrativeenheter/EnhetService.java

```diff
@@ -97,16 +143,33 @@ public void harvestAndSave() {
                             .build());
 
             enhetRepository.deleteAll();
+            enhetVariantRepository.deleteAll();
 
             final AtomicInteger counter = new AtomicInteger(0);
             final Iterable<Enhet> iterable = enhetHarvester.harvest().toIterable();
             iterable.forEach(item -> counter.getAndIncrement());
             log.info("Harvest and saving {} administrative enheter", counter.get());
             enhetRepository.saveAll(iterable);
 
+            final List<EnhetVariant> docVariants = StreamSupport.stream(iterable.spliterator(), false)
+                    .map(enh -> EnhetVariant.builder()
+                            .uri(enh.getUri().replace("/id/", "/doc/"))
+                            .name(enh.getName())
+                            .code(enh.getCode())
+                            .build())
+                    .toList();
+            enhetVariantRepository.saveAll(docVariants);
+
+            final List<EnhetVariant> idVariants = StreamSupport.stream(iterable.spliterator(), false)
+                    .flatMap(this::idVariantsOfEnhet)
+                    .toList();
+            enhetVariantRepository.saveAll(idVariants);
+
             Model model = ModelFactory.createDefaultModel();
             model.setNsPrefix("dct", DCTerms.NS);
             iterable.forEach(item -> addEnhetToModel(item, model));
+            docVariants.forEach(item -> addEnhetVariantToModel(item, model));
+            idVariants.forEach(item -> addEnhetVariantToModel(item, model));
```

Comment:
Hva er forskjellen mellom disse to?

Reply:
https://github.com/Informasjonsforvaltning/fdk-reference-data/issues/231

docVariants er hvor /id/ byttes ut med /doc/
idVariants er hvor lange ider er splitta opp til de gamle verdiene på 6 siffer